solidity/contracts/deposit/DepositUtils.sol

```diff
@@ -354,8 +350,7 @@ library DepositUtils {
     /// @dev            We cast the address to a uint256 to match the 721 standard.
     /// @return         The current deposit beneficiary.
     function depositOwner(Deposit storage _d) public view returns (address payable) {
-        IERC721 _tbtcDepositToken = IERC721(_d.TBTCDepositToken);
-        return address(uint160(_tbtcDepositToken.ownerOf(uint256(address(this)))));
+        return address(uint160(_d.tbtcDepositToken.ownerOf(uint256(address(this)))));
```

Comment:
Is there room here for an overload of `ownerAddressOf` that takes a `Deposit` and returns an `address` and does all of the weird type conversions for us as needed?

Reply:
Yeah that might be a good idea.

Reply:
Probably not worth cycles unless you're motivated, but yeah, seems cleaner.